### PR TITLE
FP-13_Create_Calendar_and_Add_Event

### PR DESCRIPTION
**Pull request by derekjpeters**
Title: Added calendar functionality and create event action for Family Promise

Description: This pull request adds a calendar to the Family Promise web application and includes the ability to create events. The calendar allows users to view and manage events in a user-friendly interface. The create event action enables users to add events to the calendar directly from the application. I have tested this functionality and verified that it works as expected.

This pull request includes the following changes:

Created a new calendar component and integrated it into the application
Added a new create event action to allow users to add events to the calendar
Tested the functionality using console log and verified that it works as expected
Please review and merge this pull request. Thank you!

### CODE REVIEW

**Review by alphaseinor — APPROVED** [review]
Future changes integrating ical or google calendar should be much easier after this.

File: src/components/pages/Calendar/RenderCalendar.js [hunk 1]
```diff
@@ -1,90 +1,164 @@
...
+  const handleModalCancel = () => {
+    setIsModalVisible(false);
+    setNewEvent({
+      title: '',
+      date: moment(),
+    });
   };
 
-  const calendars = event => {
-    console.log('The button is working');
```
Comment (publiminal) [inline src/components/pages/Calendar/RenderCalendar.js:26]: I'm one of those who likes commented code, but this line is useless now, don't you think ?

**Review by publiminal — APPROVED** [review]
@derekjpeters !
- It's a great template to go for theming and code structure for this kind of interactions. 
- Taking note also to use also moments for my own WIP in the search implementation.

File: src/components/pages/Calendar/RenderCalendar.js (hunk 1 above)
Comment (derekjpeters) [reply src/components/pages/Calendar/RenderCalendar.js:26]: Yes, it is but until we get the validation and BE tied we need a test point.

**Pull request merged**